MobileCenterCrashes/MobileCenterCrashes/Internals/Model/MSErrorAttachmentLogInternal.h

```diff
@@ -0,0 +1,28 @@
+#import <Foundation/Foundation.h>
+
+#import "MSErrorAttachmentLog.h"
+#import "MSLogWithPropertiesInternal.h"
+
+/**
+ * Error attachment log.
+ */
+@interface MSErrorAttachmentLog ()
+
+/**
+ * Error attachment identifier.
+ */
+@property(nonatomic, copy) NSString *attachmentId;
+
+/**
+ * Error log identifier to attach this log to.
+ */
+@property(nonatomic, copy) NSString *errorId;
+
+/**
+ * Checks if the object's values are valid.
+ *
+ * return YES, if the object is valid
```

Comment:
`@return` and period at the end of the sentence. Only for this file. I already addressed this in a separate PR.